packages/amplication-client/src/Resource/ResourceHome.tsx

```diff
@@ -49,12 +59,19 @@ const ResourceHome = ({ match, innerRoutes }: Props) => {
               backgroundColor:
                 resourceThemeMap[currentResource?.resourceType].color,
             }}
+            onMouseEnter={handleMouseEnter}
+            onMouseLeave={handleMouseLeave}
           >
             <ResourceNameField
+              hovered={hovered}
```

Comment:
why do you pass the set and the value to new component on props and not create them inside the component who uses them

Reply:
To get the hovered state in the Formik input for edit icon.